### PR TITLE
Cart management

### PR DESCRIPTION
**Pull request by JaanviNL**
Screenshot (38)

### CODE REVIEW

File: navigation pages/Cart/cart.css [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+    display: flex;
+    padding: 1rem 1rem 0.5rem 2rem;
+    justify-content: space-between;
+    align-items: center;
+    width: 81rem;
+    height: 5rem;
+    position: sticky;
+    top: 0;
+    left: 0;
+    z-index: 999;
```
Comment (Rohitprasad83) [inline navigation pages/Cart/cart.css:24]: can use a lower z-index value

File: navigation pages/Cart/cart.css [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+
+.logo-cart>img {
+    width: 15rem;
+    height: 4rem;
+}
+
+.input-cart {
+    width: 20rem;
+    height: 2rem;
+    border-radius: 0.1rem;
```
Comment (Rohitprasad83) [inline navigation pages/Cart/cart.css:37]: use pixel instead of 0.1rem if you want a consistent border radius

**Review by Rohitprasad83 — APPROVED** [review]
few minor changes required, rest looks good to me!

File: navigation pages/Cart/cart.html [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+    <link rel="stylesheet" href="https://explore-ui.netlify.app/style.css">
+    <script src="https://kit.fontawesome.com/d1df61a108.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="cart.css">
+    <title>Cart</title>
+</head>
+
+<body>
+    <div class="header-nav-cart">
+        <div class="logo-cart">
+            <img src="/assets/logo.png">
```
Comment (Aditi0505) [inline navigation pages/Cart/cart.html:17]: Please add an alt description.

File: navigation pages/Cart/cart.html [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+                    <div class=" horizontal-text ">
+                        <p class="card-info info">Lorem ipsum dolor sit amet consectetur .
+                        </p>
+                        <p class="sub-title info-price">Rs.340 <span class="strike">Rs.499</span> 15%</p>
+                        <div class="qty">Qty:
+                            <button class="input-btn">+</button>
+                            <input type="number" class="input-qty" value="1">
+                            <button class="input-btn">-</button>
+                        </div>
+                        <div>Size: <select name="" id="" class="drop-down">
```
Comment (Aditi0505) [inline navigation pages/Cart/cart.html:51]: Can remove id if not needed.

File: navigation pages/Cart/cart.html [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+                    <button class="card-btn btn-cart">Move To Wishlist</button>
+                    <button class="card-btn btn-cart remove-wishlist">Remove</button>
+
+
+
+                </div>
+            </div>
+            <div class="horizontal-card">
+                <div class="img-with-text">
+                    <img src="/assets/p2.webp" class="horizontal-img">
```
Comment (Aditi0505) [inline navigation pages/Cart/cart.html:71]: Can add alt description for accessibility checkpoint.

File: navigation pages/Cart/cart.html [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+                            </select></div>
+                    </div>
+                </div>
+
+                <div class="text-button">
+                    <button class="card-btn btn-cart">Move To Wishlist</button>
+                    <button class="card-btn btn-cart remove-wishlist">Remove</button>
+
+
+
```
Comment (Aditi0505) [inline navigation pages/Cart/cart.html:126]: Can remove extra spacing.

**Review by Aditi0505 — APPROVED** [review]
Left few suggestions. @Rohitprasad83 too has shared some great suggestions. 
Rest looks good to me!

File: navigation pages/Cart/cart.css [hunk 7]
```diff
@@ -0,0 +1,175 @@
+@import url(/index.css);
+@import url(/store.css);
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300&display=swap');
+
+* {
+    margin: 0;
```
Comment (ianuragsharma) [inline navigation pages/Cart/cart.css:6]: If you have reset class in index.css there is no need to repeat code here.

File: navigation pages/Cart/cart.css (hunk 2 above)
Comment (ianuragsharma) [reply navigation pages/Cart/cart.css:37]: Same vikram suggested to use pixel for value values lower than 10px

File: navigation pages/Cart/cart.css [hunk 8]
```diff
@@ -0,0 +1,175 @@
...
+}
+
+.cart-card {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    gap: 1rem;
+}
+
+h3 {
+    margin-bottom: 0.5rem;
```
Comment (ianuragsharma) [inline navigation pages/Cart/cart.css:155]: If you are using class for everything use classes here also,be consistent.

**Review by ianuragsharma — APPROVED** [review]
Suggested few changes,Please consider them

File: navigation pages/Cart/cart.css (hunk 7 above)
Comment (JaanviNL) [reply navigation pages/Cart/cart.css:6]: I haven't added reset class yet

File: navigation pages/Cart/cart.html (hunk 4 above)
Comment (JaanviNL) [reply navigation pages/Cart/cart.html:51]: Has kept it for future refrences

File: navigation pages/Cart/cart.html (hunk 3 above)
Comment (JaanviNL) [reply navigation pages/Cart/cart.html:17]: okay

**Comment by prateek1361** [comment]
looks good to me.

**Pull request merged**